sotodlib/coords/focal_plane.py

```diff
@@ -0,0 +1,280 @@
+"""
+Map positions on physical focal plane to sky using physical optics.
+Currently only works for the LAT.
+
+LAT code adapted from code provided by Simon Dicker.
+"""
+import logging
+import numpy as np
+from scipy.interpolate import interp2d
+from scipy.spatial.transform import Rotation as R
+from sotodlib import core
+
+logger = logging.getLogger(__name__)
+
+# Dictionary of zemax tube layout.
+# +ve x is to the right as seen from back of the cryostat *need to check*

```

Comment:
I am not sure about this layout, here is what I keep the record when I looked into this
https://docs.google.com/spreadsheets/d/1od2IfJo6Czt9ExH7wDADbaRPL77qi270XpjnF0f5c4U/edit#gid=0

Reply:
Interesting, there is an off by one error that I introduced for indexing purposes but even with that many of these are inconsistent. I will check with @sdicker to see what he thinks.